### PR TITLE
Remove using namespace seqan

### PR DESCRIPTION
**Pull request by joshuak94**
This can be merged independently of #8, as they touch completely different files.

### CODE REVIEW

**Comment by smehringer** [comment]
Do you actually remove the `using namespace` somewhere?

File: include/mateHunter.h [hunk 1]
```diff
@@ -10,23 +10,22 @@
 #include <config.h>
 
 using namespace std;
-using namespace seqan;
```
Comment (joshuak94) [inline include/mateHunter.h:13]: Removed here!

File: include/merge_split_alignments.h [hunk 2]
```diff
@@ -6,11 +6,9 @@
 #include <config.h>
 #include <basics.h>
 
-using namespace seqan;
```
Comment (joshuak94) [inline include/merge_split_alignments.h:9]: And here.

File: include/polishing.h [hunk 3]
```diff
@@ -15,89 +15,88 @@
 #include <mapping.h>
 
 using namespace std;
-using namespace seqan;
```
Comment (joshuak94) [inline include/polishing.h:18]: And here.

File: src/utilities_merge_split_alignments.cpp [hunk 4]
```diff
@@ -4,7 +4,6 @@
 #include <merge_split_alignments.h>
 
 using namespace std;
-using namespace seqan;
```
Comment (joshuak94) [inline src/utilities_merge_split_alignments.cpp:7]: And here is the last.

**Comment by joshuak94** [comment]
@smehringer I commented where I removed it! It was only in a few files, but I guess it is carried into the others via includes?

File: include/basics.h [hunk 5]
```diff
@@ -187,11 +187,11 @@ tuple<int, int> get_read_region_boundaries(BamAlignmentRecord const & record,
 
     if (ref_region_begin >= ref_region_end)
     {
-        cerr << "[GET READ REGION ERROR]"
+        std::cerr << "[GET READ REGION ERROR]"
              << " for record " << record.qName
```
Comment (smehringer) [inline include/basics.h]: fix indentation

File: include/basics.h [hunk 6]
```diff
@@ -229,17 +229,17 @@ tuple<int, int> get_read_region_boundaries(BamAlignmentRecord const & record,
...
+    return std::make_tuple(read_region_begin, read_region_end);
 }
 
 namespace seqan
 {
 //!\brief Overload appendValue for seqan::viewRecords, such that quality is checked
 // before appending a read
-void appendValue(std::vector<seqan::BamAlignmentRecord> & records,
+inline void appendValue(std::vector<seqan::BamAlignmentRecord> & records,
                  seqan::BamAlignmentRecord & rec)
```
Comment (smehringer) [inline include/basics.h]: indentation

File: include/basics.h [hunk 7]
```diff
@@ -255,7 +255,7 @@ void appendValue(std::vector<seqan::BamAlignmentRecord> & records,
  * @param end        The end of the region to extract flanks for.
  * @param length     The length of each flank.
  */
-seqan::Dna5String append_ref_flanks(seqan::Dna5String const & seq,
+inline seqan::Dna5String append_ref_flanks(seqan::Dna5String const & seq,
                                     seqan::FaiIndex const & fai_index,
```
Comment (smehringer) [inline include/basics.h]: indentation

File: include/basics.h [hunk 8]
```diff
@@ -292,9 +292,9 @@ seqan::Dna5String append_ref_flanks(seqan::Dna5String const & seq,
  * @param end        The end of the region to extract flanks for.
  * @param length     The length of each flank.
  */
-void records_to_read_pairs(StringSet<Dna5QString> & reads1,
-                           StringSet<Dna5QString> & reads2,
-                           std::vector<BamAlignmentRecord> & records,
+inline void records_to_read_pairs(seqan::StringSet<seqan::Dna5QString> & reads1,
+                           seqan::StringSet<seqan::Dna5QString> & reads2,
```
Comment (smehringer) [inline include/basics.h]: indentation

File: include/basics.h [hunk 9]
```diff
@@ -387,7 +387,7 @@ void records_to_read_pairs(StringSet<Dna5QString> & reads1,
     }
 }
 
-void cut_down_high_coverage(std::vector<seqan::BamAlignmentRecord> & short_reads,
+inline void cut_down_high_coverage(std::vector<seqan::BamAlignmentRecord> & short_reads,
                             int16_t mean_coverage_of_short_reads)
```
Comment (smehringer) [inline include/basics.h]: indentation

File: include/basics.h [hunk 10]
```diff
@@ -468,11 +468,11 @@ void subsample(container_type & c1, container_type & c2, unsigned N)
  * @param quals The corresponding the qualities for each sequence (position based).
  * @return  Returns the consensus sequence obtained from the MSA profile.
  */
-inline Dna5String build_consensus(StringSet<Dna5String> const & seqs,
-                                  vector<double> const & quals) // mapping qualities
+inline seqan::Dna5String build_consensus(seqan::StringSet<seqan::Dna5String> const & seqs,
+                                  std::vector<double> const & quals) // mapping qualities
```
Comment (smehringer) [inline include/basics.h]: indentation

File: include/mateHunter.h [hunk 11]
```diff
@@ -9,24 +9,21 @@
...
-
 /*! Find the mate of a read by searching in the BAM file.
  *
  */
-BamAlignmentRecord mateHunt(BamAlignmentRecord const & record,
-                             BamFileIn & bam_file,
-                             BamIndex<Bai> const & bam_index)
+inline seqan::BamAlignmentRecord mateHunt(seqan::BamAlignmentRecord const & record,
+                                   seqan::BamFileIn & bam_file,
```
Comment (smehringer) [inline include/mateHunter.h]: indentation

File: include/polishing.h [hunk 12]
```diff
@@ -208,97 +205,97 @@ void compute_baseQ_stats(SViperConfig & config,
...
+    std = std::sqrt(std / (2 * seqan::length(reads1) * seqan::length(reads1[0])));
 
     config.baseQ_mean = avg;
     config.baseQ_std = std;
 }
 
-inline Dna5String consensus_from_profile(String<ProfileChar<Dna5, double> > const & profile,
-                                         Gaps<Dna5String, ArrayGaps> const & contigGaps,
+inline seqan::Dna5String consensus_from_profile(seqan::String<seqan::ProfileChar<seqan::Dna5, double> > const & profile,
+                                         seqan::Gaps<seqan::Dna5String, seqan::ArrayGaps> const & contigGaps,
```
Comment (smehringer) [inline include/polishing.h]: indentation

File: include/polishing.h [hunk 13]
```diff
@@ -317,12 +314,12 @@ Dna5String polish(seqan::StringSet<seqan::Dna5QString> const & reads1,
     return ref;
 }
 
-Dna5String polish_to_perfection(StringSet<Dna5QString> const & reads1,
-                                StringSet<Dna5QString> const & reads2,
-                                Dna5String ref,
+seqan::Dna5String polish_to_perfection(seqan::StringSet<seqan::Dna5QString> const & reads1,
+                                seqan::StringSet<seqan::Dna5QString> const & reads2,
```
Comment (smehringer) [inline include/polishing.h]: indentation

**Review by smehringer — COMMENTED** [review]
only fix the indentation then I will merge this

**Review by smehringer — APPROVED** [review]

**Pull request merged**